editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -729,7 +730,7 @@ export function parseCode(filename: string, sourceText: string): ParsedTextFile
     let imports: Imports = emptyImports()
     // Find the already existing UIDs so that when we generate one it doesn't duplicate one
     // existing further ahead.
-    const alreadyExistingUIDs: ReadonlyArray<string> = collatedUIDs(sourceFile)
+    const alreadyExistingUIDs: Set<string> = emptySet() // collatedUIDs(sourceFile)
```

Comment:
Was this commented code intended to be deleted?

Reply:
Yes, that walks the entire file to get the existing data-uid fields but the parse will be doing that anyway so it saves some work.